playbooks/roles/streisand-gateway/templates/firewall-fr.md.j2

```diff
@@ -42,8 +42,10 @@ Si vous installez Streisand sur un serveur existant ou un fournisseur alternatif
   * TCP - {{ shadowsocks_server_port }}
   * UDP - {{ shadowsocks_server_port }}
 {% endif %}
+{% if streisand_stunnel_enabled %}
 * stunnel
   * TCP - {{ stunnel_remote_port }}
+{% endif %}
```

Comment:
This block of code should be deleted, the stunnel port is wrapped in the OpenVPN if block (line 28-39)